src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC } from 'react';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+import { Loader } from './Loader';
+
+type Props = {
+  people: Person [];
+  isLoading: boolean
+};
+
+export const PeopleTable: FC<Props> = ({ people, isLoading }) => {
+  const checkingMotherName = (personParentName: string | null) => {
```

Comment:
you may add it if you know what is it (you will learn it in future topics)
anyway, if you have any questions - feel free to ask in the chat